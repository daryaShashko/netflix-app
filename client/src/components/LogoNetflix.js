import React from 'react';

export const LogoNetflix = (props) => (
  <div className={'logo' + props.className} >
    <a href="https://netflixroulette.net/" className="logo__link"></a>
  </div>
);

LogoNetflix.defaultProps = {
  className: ''
};