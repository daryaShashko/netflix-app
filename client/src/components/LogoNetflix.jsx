import React from 'react';
import PropTypes from 'prop-types';

export const LogoNetflix = (props) => (
  <div className={'logo' + props.className} >
    <a href="https://netflixroulette.net/" className="logo__link"></a>
  </div>
);

LogoNetflix.defaultProps = {
  className: ''
};

LogoNetflix.propTypes = {
  className:  PropTypes.string
};