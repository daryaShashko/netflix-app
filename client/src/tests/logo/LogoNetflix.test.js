import React from 'react';
import ShallowRenderer from '../../../../node_modules/react-test-renderer/shallow';
import { LogoNetflix } from './../../components/LogoNetflix.jsx';

const renderer = new ShallowRenderer();

describe('LogoNetflix', () => {
  test('renders correctly', () => {
    const logo = renderer.render(<LogoNetflix />);
    expect(logo).toMatchSnapshot();
  });
});