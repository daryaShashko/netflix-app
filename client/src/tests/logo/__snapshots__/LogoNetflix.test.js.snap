// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoNetflix renders correctly 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link"
    href="https://netflixroulette.net/"
  />
</div>
`;
